src/software/ai/hl/stp/play/enemy_free_kick/enemy_free_kick_play.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "proto/parameters.pb.h"
+#include "software/ai/hl/stp/play/enemy_free_kick/enemy_free_kick_play_fsm.h"
+#include "software/ai/hl/stp/play/play.h"
+
+/**
+ * Play for defending against enemy free kicks
+ */
+class EnemyFreeKickPlay : public Play
+{
+   public:
+    EnemyFreeKickPlay(TbotsProto::AiConfig config);
```

Comment:
this constructor needs docs

Reply:
Since this is not the FSM header, do we still need constructor docs (there is already a class level comment).

Reply:
Generally, we err on more documentation than less. This is definitely not what industry typically does. But I think a constructor level comment is nice because it pops up on some editors in their code completion tool when you start calling the constructor.

Inherited functions don't need docs if they're documented in the base class.